packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -1,26 +1,21 @@
 @import '../settings/index.scss';
-
```

Comment:
Were all these new line deletes from your code editor?

Reply:
This was from my local code editor when running some linting locally.

Reply:
hm... thats weird, because i dont think the linter should have those rules, or at least I'm not getting those changes when i lint. are you running `yarn lint`?

Reply:
I think that might have been my local linter and not `yarn lint` :-/

Reply:
do you mind adding them back? it makes the spacing inconsistent in a lot of areas and it makes the PR harder to merge in general